soc/arm/st_stm32/stm32l1/Kconfig.defconfig.stm32l151xb

```diff
@@ -15,4 +15,26 @@ config NUM_IRQS
 	int
 	default 45
 
+if GPIO_STM32
+
+config GPIO_STM32_PORTA
```

Comment:
GPIOA, B and C are enabled in `st_stm32/common`

Reply:
Okay